bugbug/bug_features.py

```diff
@@ -134,6 +134,10 @@ def cleanup_fileref(text):
     return re.sub(r'\w+\.py\b|\w+\.json\b|\w+\.js\b|\w+\.jsm\b|\w+\.html\b|\w+\.css\b|\w+\.c\b|\w+\.cpp\b|\w+\.h\b', '__FILE_REFERENCE__', text)
 
 
+def cleanup_dll(text):
+    return re.sub(r'\w+\.dll', '__DLL_NAME__', text)
```

Comment:
Let's only replace when either the word ends at `.dll`, or when there is a `@` after `.dll`.